vector/src/main/java/im/vector/activity/LoginActivity.java

```diff
@@ -527,9 +527,12 @@ private void removeNetworkStateNotificationListener() {
         }
     }
 
-    @Override
-    protected void onResume() {
+    protected void onResume(Bundle savedInstanceState) {
         super.onResume();
```

Comment:
Don't you think we should call super.onResume with `savedInstanceState` too?

Reply:
Bundle can not be applied on super.onResume() method.